home.html

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link
+    rel="stylesheet"
+    href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/4.1.1/animate.min.css"
+  />
+
+    <title>Capstone project</title>
+</head>
+<!-- social section  -->
+<header class="social">
+    <img src="./images/facebook (1).png" alt="facebook" class="facebook">
+    <img src="./images/twitter (1).png" alt="twitter" class="twitter">
+
+    <ul>
+        <li><a href="#">English</a></li>
+        <li><a href="#">My Page</a></li>
+        <li><a href="#">Logout</a></li>
+    </ul>
+</header>
+<body>
+
+    <!-- mobile navigation  -->
+<div class="head">
+    <nav class="navbar">
+        <div class="navv">
+            <div class="ham">
+                <ion-icon name="menu"></ion-icon>
+                <ion-icon name="close-outline" class="close-btn"></ion-icon>
+            </div>
+            <div class="logoo">
+                <div class="logoimg"><a href="./home.html"><img src="./images/thelogo.png" alt="logo" width="100px" height="90px"></a></div>
+                <div class="text">
+                    <p>Click logo!</p>
+                </div>
+            </div>
+        </div>
+        <div class="list_nav">
+            <ul>
+                <li><a href="./home.html">Home</a></li>
+                <li><a href="./about.html">About</a></li>
+                <li><a href="#project">Project</a></li>
+                <li><a href="#join">Join</a></li>
+                <li><a href="#sponsor">Sponsor</a></li>
+                <li><a href="#news">News</a></li>
+            </ul>
+            <button type="submit" class="campaign">Learn more</button>
+        </div>
+
+        <!-- desktop version  -->
+        <div class="list_navs">
+            <ul>
+                <li><a href="./about.html">About</a></li>
+                <li><a href="#project">Project</a></li>
+                <li><a href="#join">Join</a></li>
+                <li><a href="#sponsor">Sponsor</a></li>
+                <li><a href="#news">News</a></li>
+            </ul>
+            <button type="submit" class="campaign">Learn more</button>
+        </div>
+        
+    </nav>
+     
+    <!-- hero section  -->
+    <div class="head_content">
+        <h3>"Hello Investors!"</h3>
+        <h2>Find that <br>elusive Unicorns</h2>
+    </div>
+    <div class="header_quote">
+        <p><span><strong>SANO</strong></span> will enable you to discover the newest startups to invest in. It is used by millions of people Worldwide with the best customer satisfaction.</p>
+    </div>
+    <div class="invite_date">
+        <h2><i>COME SEE US IN THE FREE WEBINER</i></h2>
+        <H3>2022:26:11(SAT) ~ 27(SUN)</H3>
+        <p>@International startups Meet, Nigeria, Abuja.</p>
+    </div>
+</div>
```

Comment:
- [ ] Make sure that the home page looks exactly like the provided design. kindly consider changing the background image to something similar to the CC design, so your page looks exactly as the provided design

|Your Page|CC design|
|-|-|
|![Screenshot_20221123_201653](https://user-images.githubusercontent.com/25227022/203629330-462d83fa-4113-49b5-b601-49f3499573d8.png)|![Screenshot_20221123_201730](https://user-images.githubusercontent.com/25227022/203629358-136bdaa6-bbc5-4092-8032-8d144a072a0d.png)|